audit-triplestore/src/main/java/org/fcrepo/camel/audit/triplestore/AuditSparqlProcessor.java

```diff
@@ -62,7 +61,9 @@
      */
     public void process(final Exchange exchange) throws Exception {
         final Message in = exchange.getIn();
-        final UriRef eventURI = new UriRef("XXX");
+        final String eventURIBase = (String) exchange.getProperty("eventURI.base");
```

Comment:
It's not particularly risky given that it's set in the EventRouter class from a property placeholder value. But wouldn't it make more sense to put the value in a header? (I realize that the distinction b/t header and property is a bit thin)